lib/shelley/cardano-wallet-shelley.cabal

```diff
@@ -0,0 +1,55 @@
+name:                cardano-wallet-shelley
+version:             2019.7.24
+synopsis:            Wallet backend protocol-specific bits implemented using Shelley nodes
+description:         Please see README.md
+homepage:            https://github.com/input-output-hk/cardano-wallet
+author:              IOHK Engineering Team
+maintainer:          operations@iohk.io
+copyright:           2019 IOHK
+license:             MIT
```

Comment:
Shouldn't we use Apache 2.0?

Reply:
Good point. Actually, we have to for all existing modules